src/block/icon-list/style.scss

```diff
@@ -35,3 +35,12 @@
 	margin-left: var(--stk-alignment-margin-left);
 	margin-right: var(--stk-alignment-margin-right);
 }
+
+//Suggestion from blocksy, prevents.
+.wp-block-stackable-icon-list ul {
+	margin-bottom: 0;
+}
+
+.wp-block-stackable-icon-list ul li:last-child {
+	margin-bottom: 0;
+}
```

Comment:
1. This should follow the class naming we use. We use `.stk-block-icon-list` instead of `.wp-block-stackable-icon-list`

2. We also use `ol` here, you may need to use `:is(ul, ol)`